package.json

```diff
@@ -37,7 +37,7 @@
   "dependencies": {
     "@algolia/cache-in-memory": "^4.14.2",
     "@algolia/recommend": "^4.12.2",
-    "@nuxt/kit": "^3.4.0",
```

Comment:
suggestion: When you would like to bump `@nuxt/kit` make sure to also bump `nuxt` version so that both will be in sync (in your PR, nuxt would remain at ^3.4.0) which can cause some unwanted issues.

Reply:
Oh you are very right! I bumped nuxt kit, because 3.4 had weird type declarations for the logger, but I forgot to update nuxt itself, sorry ><

Reply:
Done!

Reply:
Thanks!